<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fileAddition</class>
 <widget class="QWidget" name="fileAddition">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>217</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <property name="columnCount">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QToolButton" name="btnAdd">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>btnAdd</string>
         </property>
         <property name="icon">
          <iconset resource="img.qrc">
           <normaloff>:/file/Resources/file/add.png</normaloff>:/file/Resources/file/add.png</iconset>
         </property>
         <property name="shortcut">
          <string/>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnRemove">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::DefaultContextMenu</enum>
         </property>
         <property name="text">
          <string>btnRm</string>
         </property>
         <property name="icon">
          <iconset resource="img.qrc">
           <normaloff>:/file/Resources/file/cut_inactive.png</normaloff>:/file/Resources/file/cut_inactive.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnUp">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>btnUp</string>
         </property>
         <property name="icon">
          <iconset resource="img.qrc">
           <normaloff>:/file/Resources/file/up_inactive.png</normaloff>:/file/Resources/file/up_inactive.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnDown">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>btnDown</string>
         </property>
         <property name="icon">
          <iconset resource="img.qrc">
           <normaloff>:/file/Resources/file/down_inactive.png</normaloff>:/file/Resources/file/down_inactive.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
